Common/tests/TestBase/ResultTableTestBase.cs

```diff
@@ -750,4 +760,39 @@ public void AddDependentNotExistingResultShouldThrow()
                                                                                 .ConfigureAwait(false));
     }
   }
+
+  [Test]
+  public async Task SetTaskOwnershipShouldSucceed()
```

Comment:
Would it make sense to do this test on both a result with and without an ownerTaskId?

Reply:
If we override the value, it does not make any difference if there is a value or not. I need to check what is the exact query

Reply:
The test I propose would make sure this assumption is correct. I agree the current implementation do not care (and must not care), but I can imagine that some implementation might have make an invalid assumption.
I'm fine both ways, though.